cli/app-module-template/middlewares/__#{name}-middleware.js

```diff
@@ -0,0 +1,6 @@
+const #{camelCaseName}Middleware = store => next => action => {
```

Comment:
Solely mine misunderstanding of `store => next => action => {}`.

Reply:
?

Reply:
Just my personal feeling about code readability...

Reply:
Not yours alone 😆 
But that's the way they say to do it, we have nothing to do, just accept this reality 😢

Reply:
But.... https://github.com/21-23/_qd-ui/blob/master/app/game/middleware/server-pipe.js#L137..L142

Reply:
In this case I'm just following community I trust to :) [airbnb-styleguide](https://github.com/airbnb/javascript#arrows--implicit-return)

Reply:
Right, but they forgot about an example of function that returns function that returns function :) Pay attention to their description: `Why? Syntactic sugar. It reads well when multiple functions are chained together.`
Examples like 

```
[1, 2, 3].map(number => `A string containing the ${number}.`);
```

look OK (even though I don't really like it because it (was) impossible to set the breakpoint).

Reply:
they forgot to add example, but their eslint rules will force this exact style

Reply:
This style has already grown into a habit of mine since that's how I achieve curried functions when I tell the students about FP... Looks quite odd amidst more "conventional" JS function declarations though.